sources/settings-plugins/aws-k8s-nvidia/src/lib.rs

```diff
@@ -0,0 +1,28 @@
+use bottlerocket_settings_models::model_derive::model;
+use bottlerocket_settings_plugin::SettingsPlugin;
+
+#[derive(SettingsPlugin)]
+#[model(rename = "settings", impl_default = true)]
+struct AwsK8sSettings {
+    motd: bottlerocket_settings_models::MotdV1,
+    kubernetes: bottlerocket_settings_models::KubernetesSettingsV1,
+    updates: bottlerocket_settings_models::UpdatesSettingsV1,
+    host_containers: bottlerocket_settings_models::HostContainersSettingsV1,
+    bootstrap_containers: bottlerocket_settings_models::BootstrapContainersSettingsV1,
```

Comment:
Shouldn we be adding bootstrap-commands to nvidia variants?

Reply:
Good call out, thanks!